src/dtos/http_request.go

```diff
@@ -0,0 +1,15 @@
+package dtos
+
+import "github.com/Real-Dev-Squad/wisee-backend/src/models"
+
+type CreateUpdateFormRequestDto struct {
+	Status        string             `json:"status"`
```

Comment:
- why does frontend have to send the status? By default when creating a form it will be in the draft state, there will also be a button to `publish` the form which will change the status to `published` accordingly. 
[erd for reference](https://dbdiagram.io/d/wisee-65c11620ac844320ae7dabf5)

Reply:
I had a different user story in mind, when doing this, will change.

Reply:
My bad, the status is there for PATCH request, but yes will have to make the field nullable.